paper/ms.tex

```diff
@@ -1645,7 +1643,9 @@ \section{Ensuring positive definiteness}
 Section 4.2.1 in][]{Rasmussen:2006}~--~to requiring that the kernel be the
 Fourier transform of a positive finite measure.
 This means that the power spectrum of a positive definite kernel must be
-positive for all frequencies.
+positive for all frequencies.  This seems true intuitively:  since the
+power spectrum is the square of the Fourier transform of the data time
+series, it must be non-negative.
```

Comment:
This is exactly the language I want to avoid - the power spectrum is *not* the squared amplitude of the Fourier transform of the time series! That's just an _estimator_ of the PSD!!